cmd/main.go

```diff
@@ -77,6 +77,7 @@ func main() {
 		QueueID:                    queueID,
 		NodeName:                   nodeName,
 		NetfilterBug1766Fix:        netfilterBug1766Fix,
+		NFTableName:                "kube-network-policies-different",
```

Comment:
@tssurya if my theory is correct, the problem is that kindnet and kube-network-policies are using the same table and stomping into each other

cc: @danwinship